lib/pages/reader-page.js

```diff
@@ -12,24 +12,34 @@ export default class ReaderPage extends BaseContainer {
 		}
 		super( driver, by.css( '.is-section-reader' ), visit, url );
 	}
+
 	siteOfLatestPost() {
-		return this.driver.findElement( by.css( 'article.reader__card .site__domain' ) ).getText();
+		return this.driver.findElement( by.css( '.reader-visit-link' ) ).getAttribute( 'href' ).then( ( href ) => {
+			return href.split( '/' )[2];
```

Comment:
This works and I don't think it would be fragile, but it can be a little hard to understand what it's doing at a glance.
You could try something like:
`return URL.parse( href ).hostname;`
(which requires this also `import URL from 'url';`) which is the in built URL parsing library: https://www.w3schools.com/nodejs/nodejs_url.asp
Saying that, I've probably written very similar or the same split code as you previously so I'm happy with that too